locustempus/templates/main/course_list.html

```diff
@@ -10,14 +10,17 @@
 
 {% block page_actions %}
 <div class="lt-dashboard_actions">
-    <button type="button" class="btn lt-button mr-1 lt-button--transparent">
-        <span class='lt-icons lt-button__icon'><img src="{{STATIC_URL}}img/icon-grid.svg" class="active" /></span>
-        <span class='lt-button__text'>Grid</span>
-    </button>
-    <button type="button" class="btn lt-button lt-button--transparent">
-        <span class='lt-icons lt-button__icon'><img src="{{STATIC_URL}}img/icon-list.svg" class="inactive" /></span>
-        <span class='lt-button__text'>List</span>
-    </button>
+    <form method="post">
+        {% csrf_token %}
+        <button type="submit" class="btn lt-button mr-1 lt-button--transparent" {% if course_list_grid %}disabled{% endif %}>
```

Comment:
On the Workspaces page, when I click "Grid", which is highlighted, both "Grid" and "List" toggles disappear.